daisy/workflow/step_create_instances.go

```diff
@@ -244,11 +243,20 @@ func (c *CreateInstances) validate(s *Step) error {
 		errs = append(errs, validatedInstances.add(w, ci.daisyName))
 	}
 
-	var multiErr *multierror.Error
-	if len(errs) > 0 {
-		multiErr = multierror.Append(errs[0], errs[1:]...)
+	errStrs := []string{}
+	for _, e := range errs {
+		if e != nil {
+			errStrs = append(errStrs, e.Error())
+		}
+	}
+	if len(errStrs) > 0 {
+		msg := fmt.Sprintf("%d errors:\n", len(errStrs))
+		for _, s := range errStrs {
+			msg += fmt.Sprintf("\n* %s", s)
```

Comment:
This will add an extra line at the first error, I think you want msg += fmt.Sprintf("* %s\n", s)